cli/dockerutils.py

```diff
@@ -32,3 +34,28 @@ def docker_build(self, image_tag, docker_file):
     def docker_push(self, image):
         exit_code = os.system("docker push {0}".format(image))
         return exit_code
+
+    def docker_search_v1(self, registry, name, application):
+        result = subprocess.check_output("docker search {0}/{1}-{2}".format(
```

Comment:
We should put the code in try..except block to handle if at all any exception occurs. As well as "raise" it.